lec06-pop-models.Rmd

```diff
@@ -2,3 +2,298 @@
+If your calculus is foggy in your memory, when you see this notation think "change in $N$ per change in $t$.
+$dN$ is the notation commonly used to mean "a small amount of $N$, so this expression mathematically describes
+by what small amount $N$ will change in a small amount of time $dt$. This is also sometimes written as
+$\dot{N}$. I will usually use $dN/dt$ but you may see the dot notation in the literature. 
+A single dot is shorthand for "first derivative". 
+
+Biologically, $dN/dt$ is the **growth rate** or **rate of change** of a population with $N$ organisms at time $t$. 
+
+What should the growth rate of our population depend on? Let's brainstorm some things. 
+There are no wrong answers here: anything you can think of that makes biological sense is a reasonable thing 
+to include in a model, although after brainstorming we will write something very simple as an example.
+
+Possible answers:
+- birth rate $r$
+- death rate $\delta$
+- number of organisms $N$
+- carrying capacity $K$
+- rate of predation
+- migration rate
+- abundance of food
+
+Let's make some sample data to help us come up with a simple model.
+
+For bacteria growing in a flask with plenty of food, we know that they divide in two periodically. 
+If we take one unit of time to be the time between divisions, we can assume that the population 
+approximately doubles at each time point.
+
+```{r}
+times = seq(0, 10, by = 0.2)
+population_size <- 2^times
+plot(times,population_size)
+```
+
+How do we write this assumption as a general model? Let's calculate the difference in $N$ at each time point
+and plot this vs. $N$ at the previous time point. 
+
+```{r}
+
+N_diff <- tail(population_size,-1) - head(population_size,-1) # last N-1 points minus first N-1 points
+plot(head(population_size,-1),N_diff)
+lines(head(population_size,-1),N_diff)
+```
+This is a straight line, which tells us that the change in $N$ is proportional to $N$, 
+and in fact because of the way we defined time, the change in $N$ is exactly $N$. 
+
+Now we have our model: this model describes **exponential** growth of a population and is commonly used 
+for microbes growing with plenty of food. 
+
+$$\frac{dN}{dt}=N$$
+
+This is a **differential equation**: an equation that describes how some function of $N$ is related to
+derivatives of $N$. (In this case, and probably in this entire course, we'll just be using the first derivative.)
+
+If we wanted time to be something else, like minutes instead of generations, 
+we could add a constant of proportionality that captures how often our bacteria divide:
+
+$$\frac{dN}{dt} = rN$$
+
+This says that the population will increase by $r N dt$ in the small time $dt$, 
+or the population increases by $N dt$ in time $dt/r$. 
+
+The process we just used to get this model is probably not what you'd do in practice - 
+I just wanted to illustrate the connection between exponential growth and this differential equation.
+
+### Individual exercise
```

Comment:
Not sure if we want to be consistent with Joel's "Challenge" sections. Up to you. I do think it should be blockquoted like Joel started it.